test/mavsdk_tests/process_helper.py

```diff
@@ -32,6 +32,7 @@ def __init__(self,
         self.log_dir = log_dir
         self.log_filename = ""
         self.wait_until_complete = False
+        self.stop_thread: Any[threading.Event] = None
```

Comment:
What actually sets this `stop_thread` variable? I'm not familiar with Python threading...

Reply:
It's an event that can transfer information from one thread to another. In this case I use it to signal the thread to stop, so it can be joined.